web.config

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Platform config can be done as a separate PR.This way it would be very clear to understand.